keyboards/bm68rgb/readme.md

```diff
@@ -9,12 +9,13 @@ A 65% hotswap in switch RGB keyboard from KPRepublic.
 * Hardware Availability: [KP Republic](https://kprepublic.com/products/bm68rgb-bm68-rgb-65-hot-swappable-custom-mechanical-keyboard-pcb-programmed-qmk-via-firmware-full-rgb-switch-underglow-type-c)
 
 Make example for this keyboard (after setting up your build environment):
-
-    make bm68rgb:default
-
+```sh
+qmk compile -kb bm68rgb -km default
+```
```

Comment:
```suggestion

    make bm68rgb:default

```

Reply:
@fauxpark Is there a reason to prefer `make` over the `qmk` script for building firmware?

Reply:
For the moment, it's what's in the template.